src/util/verify-chromium-port.js

```diff
@@ -0,0 +1,52 @@
+import { createServer } from 'node:http';
+
+import { UnusablePortError } from '../errors.js';
+
+/**
+ * Determine if a port is available
+ * @param {number} port Port to test
+ * */
+async function isPortAvailable(port) {
+  return new Promise((resolve) => {
+    const server = createServer();
+    server.once('listening', () => {
+      server.close(() => resolve(true));
+    });
+    server.once('error', () => resolve(false));
+    server.listen(port);
+  });
+}
+
+/**
+ * Validate that requested port is a valid port
+ * @param {number} port Debugging port
+ * @returns {boolean}
+ */
+function isPortValid(port) {
+  if (!port) {
+    return false;
+  }
```

Comment:
```suggestion
```

The caller is expected to pass a port, this branch is unreachable. For simplicity let's just drop it (an error would be raised if `port` is somehow omitted).